embedded-localstack/src/test/java/com/playtika/test/localstack/EmbeddedLocalStackTest.java

```diff
@@ -0,0 +1,126 @@
+package com.playtika.test.localstack;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Optional;
+
+import com.amazonaws.auth.AWSCredentialsProvider;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.Bucket;
+import com.amazonaws.services.s3.model.ObjectListing;
+import com.amazonaws.services.s3.model.S3Object;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+import com.amazonaws.services.sqs.model.CreateQueueResult;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.testcontainers.DockerClientFactory;
+import org.testcontainers.shaded.org.apache.commons.io.IOUtils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = EmbeddedLocalStackTest.TestConfiguration.class,
+        properties = "embedded.localstack.services=S3,SQS")
+public class EmbeddedLocalStackTest {
+    @Value("${embedded.localstack.accessKey}")
+    private String accessKey;
+
+    @Value("${embedded.localstack.secretKey}")
+    private String secretKey;
+
+    @Value("${embedded.localstack.S3}")
+    private String s3Endpoint;
+
+    @Value("${embedded.localstack.SQS}")
+    private String sqsEndpoint;
+
+    @Value("${embedded.localstack.region}")
+    private String region;
+
+    @Value("${embedded.localstack.SQS.port}")
+    private String sqsPort;
+
+    @Autowired
+    private ConfigurableEnvironment environment;
+
+    @Test
+    public void s3TestOverBridgeNetwork() throws IOException {
+        AmazonS3 s3 = AmazonS3ClientBuilder
```

Comment:
- https://cloud.spring.io/spring-cloud-static/spring-cloud-aws/2.2.1.RELEASE/reference/html/
- https://github.com/spring-cloud/spring-cloud-aws
- https://spring.io/projects/spring-cloud-aws

\+ it would be nice to have dependsOn post processor as well